modules/core/src/test/java/org/apache/ignite/internal/GridMBeansTest.java

```diff
@@ -52,6 +52,10 @@ public GridMBeansTest() {
      * This implementation registers adds custom executors to the configuration.
      */
     @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
+        System.setProperty(IGNITE_MBEAN_APPEND_CLASS_LOADER_ID, "false");
```

Comment:
Could you please set the properties back to the initial values after the test completes?
I'd suggest to override `setUp()` and `tearDown()` methods for that.